music/admin.py

```diff
@@ -17,4 +27,18 @@ def add_music(self, request, queryset):
             music.room.add_music(**MusicSerializer(music).data)
         return
 
+    def duplicate_music(self,request,queryset):
+        targetRoom = Room.objects.get(name=request.POST['nrroom'])
+        if targetRoom:
+            for music in queryset:
+                if targetRoom.name != music.room.name:
+                    new_entry = music
```

Comment:
Actually, you just steal the music, fucking pirate hacker ! Would you steal a car ? NO ! Would you steal a TV ? NO ! So why do you steal music ????